src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -1588,7 +1588,7 @@ SELECT histogram(duration, 10, 20) FROM PageView SINCE 1 week ago
     id="func-minOf"
     title={<InlineCode>minuteOf(attribute)</InlineCode>}
   >
-    Use the minuteOf() function to extract only the minute portion (that is, seconds 0 to 59) of an attribute holding a valid timestamp value.
+    Use the `minuteOf( )` function to extract only the minute portion (that is, seconds 0 to 59) of an attribute holding a valid timestamp value.
```

Comment:
@austin-schaefer are you sure you want to insert a space between these parentheses? In the Microsoft Manual of Style, I'm not seeing examples of spaces. For example, the main function is written like this: `main()`. See https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/f/function

Plus, I'm seeing other function references in the doc that don't have this space between the parentheses.